graylog2-web-interface/src/views/components/aggregationwizard/elementConfiguration/Sort.tsx

```diff
@@ -121,4 +121,4 @@ const Sort = ({ index }: Props) => {
   );
 };
 
-export default Sort;
+export default React.memo(Sort);
```

Comment:
Same tiny nitpick as above, wrapping declaration with `React.memo` instead of export?